docs/concepts/using-the-sdk/simulation.md

```diff
@@ -0,0 +1,72 @@
+# Cosmos Blockchain Simulator
+
+The Cosmos SDK offers a full fledged simulation framework to fuzz test every message defined by a module.
+
+This functionality is provided by the[`SimApp`](https://github.com/cosmos/cosmos-sdk/blob/master/simapp/app.go),
+which is a dummy application that is used for running the [`simulation`](https://github.com/cosmos/cosmos-sdk/tree/master/x/simulation) module.
+This module defines all the simulation logic as well as the operations for randomized parameters like accounts, balances etc.
+
+## Goals
+
+The blockchain simulator tests how the blockchain application would behave under real life circumstances by generating and sending randomized messages.
+The goal of this is to detect and debug failures that could halt a live chain, by providing logs and statistics about the operations run by the simulator as well as exporting the latest application state when a failure was found.
+
+Its main difference with integration testing is that the simulator app allows you to pass parameters to customize the chain that's being simulated.
+This comes in handy when trying to reproduce bugs that were generated in the provided operations (randomized or not).
+
+## Simulation commands
+
+The simulation app has different commands, each of it tests a different failure type:
```

Comment:
"The simulation app has different commands, each of `which` tests a different failure type: